components/Templates/withFrontData.js

```diff
@@ -1,7 +1,7 @@
 import { graphql } from 'react-apollo'
 import gql from 'graphql-tag'
-import { TeaserActiveDebates } from '@project-r/styleguide/lib/components/TeaserActiveDebates'
-import { TeaserMyMagazine } from '@project-r/styleguide/lib/components/TeaserMyMagazine'
+import { TeaserActiveDebates } from '@project-r/styleguide/lib/components/TeaserActiveDebates' // should this be such?
+import { TeaserMyMagazine } from '@project-r/styleguide/lib/components/TeaserMyMagazine' // ditto
```

Comment:
for the time being let's leave it as is. eventually we can properly export it, styleguide needs a setup/tooling modernisation soon

```suggestion
import { TeaserActiveDebates } from '@project-r/styleguide/lib/components/TeaserActiveDebates'
import { TeaserMyMagazine } from '@project-r/styleguide/lib/components/TeaserMyMagazine'
```